cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -122,9 +122,12 @@
     protected String headlessServiceName;
     protected String name;
 
-    protected static final int METRICS_PORT = 9404;
-    protected static final String METRICS_PORT_NAME = "metrics";
-    protected boolean isMetricsEnabled;
+    protected static final int PROMETHEUS_METRICS_PORT = 9404;
+    protected static final String PROMETHEUS_METRICS_PORT_NAME = "prometheus";
+    protected boolean isPrometheusMetricsEnabled;
+
+    protected static final int JMX_PORT = 9999;
```

Comment:
Out of curiosity - is this random choice? Or does it have some logic? (I know that 9404 as the original JMX port is already taken)

Reply:
I chose to use 9999 because a lot of the tutorials online used 9999 as the JMX metrics port they exposed. For example in this tutorial (https://docs.oracle.com/javase/tutorial/jmx/remote/jconsole.html).

Reply:
Ok, great. Thanks.